modules/ROOT/pages/7.8.0-release-notes.adoc

```diff
@@ -69,6 +69,19 @@ The {productname} {release-version} release includes an accompanying release of
 
 For information on the **<Premium plugin name 1>** plugin, see: xref:<plugincode>.adoc[<Premium plugin name 1>].
 
+=== Comments
+
+The {productname} {release-version} release includes an accompanying release of the **Comments** premium plugin.
+
+**Comments** includes the following addition.
+
+=== New `tinycomments_fetch_author_info` option that allows integrators to specify the current author info in callback mode.
+// #TINY-11680
+
+In {productname} {release-version}, a new `+tinycomments_fetch_author_info+` callback option has been introduced, enabling integrators to provide author information in callback mode. This enhancement ensures accurate author data is displayed in new conversation cards. Additionally, author data is now pre-fetched during comment initialization to ensure immediate availability.
```

Comment:
`Additionally, author data is now pre-fetched during comment initialization to ensure immediate availability.` I don't think this part is necessary, this is just a technical detail.

```suggestion
In {productname} {release-version}, a new `+tinycomments_fetch_author_info+` callback option has been introduced, enabling integrators to provide author information in callback mode. This enhancement ensures accurate author data is displayed in new conversation cards.
```